README.md

```diff
@@ -634,6 +642,63 @@ tags: |
   type=raw,value=mytag-{{branch}}-{{sha}}
 ```
 
+#### `{{branch}}`
+
+Returns the branch name that triggered the workflow run. Will be empty if not 
+a branch reference:
+
+| Event           | Ref                           | Output              |
+|-----------------|-------------------------------|---------------------|
+| `pull_request`  | `refs/pull/2/merge`           |                     |
+| `push`          | `refs/heads/master`           | `master`            |
+| `push`          | `refs/heads/my/branch`        | `my-branch`         |
+| `push tag`      | `refs/tags/v1.2.3`            |                     |
+
+#### `{{tag}}`
+
+Returns the tag name that triggered the workflow run. Will be empty if not a
+tag reference:
+
+| Event           | Ref                           | Output             |
+|-----------------|-------------------------------|--------------------|
+| `pull_request`  | `refs/pull/2/merge`           |                    |
+| `push`          | `refs/heads/master`           |                    |
+| `push`          | `refs/heads/my/branch`        |                    |
+| `push tag`      | `refs/tags/v1.2.3`            | `v1.2.3`           |
+
+#### `{{sha}}`
+
+Returns the short commit SHA that triggered the workflow run (e.g., `90dd603`).
+
+#### `{{base_ref}}`
+
+Returns the base ref or target branch of the pull request that triggered the
+workflow run. Will be empty for a branch reference:
+
+| Event           | Ref                           | Output             |
+|-----------------|-------------------------------|--------------------|
+| `pull_request`  | `refs/pull/2/merge`           | `master`           |
+| `push`          | `refs/heads/master`           |                    |
+| `push`          | `refs/heads/my/branch`        |                    |
+| `push tag`      | `refs/tags/v1.2.3`            | `master`           |
+
+#### `{{is_default_branch}}`
+
+Returns `true` if the branch that triggered the workflow run is the default
+one, otherwise `false`.
```

Comment:
@crazy-max , could you clarify here, whether a pull-request to default branch will return true or only a `push` event to default branch will be `true`?